src/jsdocs/definitions.yaml

```diff
@@ -0,0 +1,11 @@
+definitions:
+  # BadRequestResponse model
+  BadRequestResponse:
+    type: object
+    properties:
+      status:
+        type: string
+        example: error
+      error:
+        type: string

```

Comment:
I am guessing the idea here you are trying to achieve is reusability? In that light I think this file should be more appropriately named. E.g 400-responseModel.yaml or badRequestResponseModel.yaml. That way it is obvious at a glance that this is a model. Better still reusable models can be maybe kept in a sub-folder e.g models. So anyone willing to reuse can pick definitions from there